pkg/resource/logging-agents-toggle/observability_bundle_configmap.go

```diff
@@ -43,12 +44,13 @@ func GenerateObservabilityBundleConfigMap(ctx context.Context, lc loggedcluster.
 		appsToEnable[promtailAppName] = app{
 			Enabled: true,
 		}
-		appsToEnable["alloy-logs"] = app{
+		appsToEnable[common.AlloyLogAgentName] = app{
 			Enabled: false,
 		}
-	case "alloy-logs":
-		appsToEnable["alloy-logs"] = app{
-			Enabled: true,
+	case common.AlloyLogAgentName:
+		appsToEnable[common.AlloyLogAgentName] = app{
+			Enabled:   true,
+			Namespace: common.AlloyLogAgentNamespace,
```

Comment:
Since this secret needs to live in the same namespace as Alloy pods, I am defining Alloy app namespace as a constant and using this for both creating the secret but also making sure Alloy is deployed in the given namespace in the logging-toggle resource.